File: Server/FIFA.Server/Models/MatchResultDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace FIFA.Server.Models
{
    public class MatchResultDTO
    {
        public int HomePlayerId { get; set; }
        public int AwayPlayerId { get; set; }
        public int ScoreHome { get; set; }
        public int ScoreAway { get; set; }
        public DateTime Date { get; set; }
    }
}

File: Client/typescript/module/results/MatchResultDTO.ts
module FifaLeagueClient.Module.Results {
	export class MatchResultDTO {
		public homePlayerId: number;
		public awayPlayerId: number;
		public scoreHome: number;
		public scoreAway: number;
		public date: Date;

		contructor(data){
			if (data != null){
				this.homePlayerId = data.homePlayerId;
				this.awayPlayerId = data.awayPlayerId;
				this.scoreHome = data.scoreHome;
				this.scoreAway = data.scoreAway;
				this.date = data.date;
			}
		}
	}
}